### ISSUE TITLE
Eye dropper color picker: preview feature doesn't work on latest Edge browser

### ISSUE THREAD

**Issue by nick-hoang** [umbraco/Umbraco-CMS#14627]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

12.0.1

### Bug summary

Works well on Firefox but doesn't work on Edge: https://prnt.sc/pfpN2-nebtPv

### Steps to reproduce

1. Open backoffice in latest Edge browser
2. Create a Data type named **Eye Dropper Color Picker** using **Umbraco.ColorPicker.EyeDropper** property editor
3. Create a content type named **Home** with a property named **Text color**, using data type **Eye Dropper Color Picker** above (allow as root if needed)
4. Create a node from **Home** content type then click to the color picker to pick a color in **Text color** property
5. No preview be show after selected a color: https://prnt.sc/pfpN2-nebtPv

### Expected result / actual result

5. The selected color should be show in the preview section: https://prnt.sc/8YmUnXEIY-3z

**Comment by nick-hoang** [comment]
Temporary solution: create a custom package with a single css file with this content:
```
.sp-alpha-inner, .sp-preview-inner, .sp-thumb-inner {
    height: 20px;
}
```

**Comment by bjarnef** [comment]
I am pretty sure this has already been fixed 🙂
https://github.com/umbraco/Umbraco-CMS/issues/14553

**Comment by kjac** [comment]
Hi @nick-hoang,

Thank you for reaching out 😄 

This has indeed been fixed and will be part of 11.4.3 and 12.1.0 - see Andreas' comment here: https://github.com/umbraco/Umbraco-CMS/issues/14553#issuecomment-1635461384
